src/extensions/slack/index.ts

```diff
@@ -487,21 +487,22 @@ emitter.on('sessionUpdate', async (update: {
         //     }
         // }
 
+        const elapsed = getElapsed(session);
         if (session.paused) {
             if (updateSlack) {
                 await updateController(session);
```

Comment:
Is there a reason to update the controller twice when paused? It's already updating after the if

Reply:
Probably not, although this was not introduced in this PR.

Reply:
the controller doesn't update twice (the function returns before it can make that second return call) although it's unnecessary to do so here. I'll remove it in another PR